src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -2369,10 +2389,38 @@ export class PostgresStorageAdapter implements StorageAdapter {
           sort !== undefined && sorting.length > 0 ? `ORDER BY ${sorting}` : '';
       }
     }
-
-    const qs = `SELECT ${columns.join()} FROM $1:name ${wherePattern} ${sortPattern} ${limitPattern} ${skipPattern} ${groupPattern}`;
+    const originalQuery = `SELECT ${columns.join()} FROM $1:name ${wherePattern} ${sortPattern} ${limitPattern} ${skipPattern} ${groupPattern}`;
+    const qs = explain ? this.createExplainableQuery(originalQuery) : originalQuery;
     debug(qs, values);
     return this._client
+      /*.any(qs, values)
```

Comment:
There's a bug here... Was trying to switch the mapping method to many so it will return the "explain" results here, but had trouble with converting to map/promise

Reply:
Fixed